docs/_data/repoStatus.ts

```diff
@@ -36,6 +36,8 @@ export interface RepoStatusRecord {
   name: string;
   /** Appears on the header of the docs page */
   description?: string;
+  /** Appears above the header on the docs page */
+  preamble?: string;
```

Comment:
let's just put this in the docs file itself, rather than add config here

Reply:
@bennypowers the goal was to get it above the `Overview` which is added automatically.

Reply:
I'd rather not maintain more members in repoStatus, and I certainly don't want to have to trace the threads to find docs content in repoStatus.ts

Can this be a persistent alert, or an inline alert after the heading? that's attention grabbing, consistent, and doesn't require unrelated changes to our docs framework.